src/objectives/elbo/entropy.jl

```diff
@@ -0,0 +1,34 @@
+
+struct ClosedFormEntropy <: AbstractEntropyEstimator end
+
+function (::ClosedFormEntropy)(q, ::AbstractMatrix)
+    entropy(q)
+end
+
+struct MonteCarloEntropy <: AbstractEntropyEstimator end
+
+function (::MonteCarloEntropy)(q, ηs::AbstractMatrix)
+    mean(eachcol(ηs)) do ηᵢ
+        -logpdf(q, ηᵢ)
+    end
+end
+
+"""
+    StickingTheLandingEntropy()
+
+The "sticking the landing" entropy estimator.
+
+# Requirements
+- `q` implements `logpdf`.
+- `logpdf(q, η)` must be differentiable by the selected AD framework.
+
+# References
+* Roeder, G., Wu, Y., & Duvenaud, D. K. (2017). Sticking the landing: Simple, lower-variance gradient estimators for variational inference. Advances in Neural Information Processing Systems, 30.
+"""
+struct StickingTheLandingEntropy <: AbstractEntropyEstimator end
+
+function (::StickingTheLandingEntropy)(q, ηs::AbstractMatrix)
+    ChainRulesCore.@ignore_derivatives mean(eachcol(ηs)) do ηᵢ
```

Comment:
Does this work for ForwardDiff?

Reply:
Oh... I guess this will have to go to `ext`. This explains why some of the tests involving STL have been failing on lower tolerance levels!

Reply:
Reverted back to the "manual stopping" approach since it seems not all AD backends provide a stopping mechanism (ForwardDiff in particular unless I missed it?)